omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -363,6 +363,14 @@ function PatientSearchWidget(configuration){
                             }
                         });
                     }
+                    else if (column.type === 'identifierLocation') {
+                        jq.each(patient.identifiers, function (index, patientIdentifier) {
+                            var identifierLocation = patientIdentifier.location;
+                            if (identifierLocation != null) {
+                                columnValue = identifierLocation.display
```

Comment:
change to:  (the part in the in middle makes sure we add a comma between each value)

// create a string of all matching identifier values
columnValue = columnValue + (columnValue !== '' ? ', ' : '') + identifierLocation.display